### PR TITLE
Calypso build: Add changelog and bump to v2

### PR DESCRIPTION
**Pull request by sirreal**
#### Changes proposed in this Pull Request

* Bump the package version for release. See new changelog for details.

#### Testing instructions

* No code changes.

### CODE REVIEW

File: client/components/screen-reader-text/style.scss [hunk 1]
```diff
@@ -1,5 +1,5 @@
 // Text meant only for screen readers
-.screen-reader-text { // TODO add _accessibility.scss
```
Comment (sirreal) [inline client/components/screen-reader-text/style.scss]: Completely unrelated, but I wanted to remove this todo.

It's confusing https://github.com/Automattic/jetpack/pull/12277#discussion_r282027675

And it's ancient: 1075-gh-calypso-pre-oss

I think it's safe to say that it's not a priority and no one is coming back to do it.

**Review by dmsnell — DISMISSED** [review]
Can you explain more what the implications are for consumers of this build tool? Will code that was previously fine in v1 now break the v2 build?

What do people need to change in order to upgrade the version?

**Comment by sirreal** [comment]
> Can you explain more what the implications are for consumers of this build tool? Will code that was previously fine in v1 now break the v2 build?
> 
> What do people need to change in order to upgrade the version?

If folks were compiling code that relied on the removed babel transforms (export default from/export namespace from), they'd need to make changes. Basically the breaking changes correspond to what was changed in #32871.

**Comment by dmsnell** [comment]
> If folks were compiling code that relied on the removed babel transforms…

I meant in the CHANGELOG. Maybe we wouldn't do that, but it seems like it would be awfully helpful to have a simple example showing the important changes.

**Comment by sirreal** [comment]
I added more details to the v2 changelog.

**Review by dmsnell — APPROVED** [review]
Those explanations are so incredibly helpful - thanks @sirreal

**Pull request merged**